src/components/views/rooms/RoomListPanel/RoomListOptionsMenu.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+import { IconButton, Menu, MenuTitle, CheckboxMenuItem, Tooltip } from "@vector-im/compound-web";
+import React, { forwardRef, type JSX, useState } from "react";
+import OverflowHorizontalIcon from "@vector-im/compound-design-tokens/assets/web/icons/overflow-horizontal";
+
+import { _t } from "../../../../languageHandler";
+import { type RoomListViewState } from "../../../viewmodels/roomlist/RoomListViewModel";
+
+const MenuTrigger = forwardRef<HTMLButtonElement>((props, ref) => (
```

Comment:
We're in React 19 now, ref is just a normal prop, `forwardRef` is no longer needed https://react.dev/blog/2024/12/05/react-19#improvements-in-react-19